lib/AppInfo/Application20.php

```diff
@@ -84,6 +85,12 @@ public function boot(IBootContext $context): void {
 		$notificationManager = $context->getServerContainer()->get(\OCP\Notification\IManager::class);
 		$notificationManager->registerNotifierService(Notifier::class);
 		\OCP\Util::addStyle('deck', 'deck');
+
+		$this->registerUserGroupHooks();
+
+		$this->registerCommentsEntity();
```

Comment:
tip: use ``\OCP\AppFramework\Bootstrap\IBootContext::injectFn`` so you don't have to locate services manually but have them injected as args :)

Reply:
Oh nice :+1: